src/pages/sessions/[id]/controls.tsx

```diff
@@ -16,23 +17,27 @@ const SessionControlsPage: NextPage = () => {
   const session = useTeamLobbySubscription(userId);
   const endSession = useUpdateTeamStatus(teamId, 'ENDED');
 
+  const activeUsers = useOnlineUsers(teamId);
+  const onlineUsers = activeUsers ? activeUsers.online_team_users : [];
+
   const usersWithEntry = useMemo(
-    () => session?.user_by_pk.team.entries.map((item) => item.user.id),
-    [session?.user_by_pk.team.entries.map],
+    () => onlineUsers.map((item) => item.id),
+    [onlineUsers],
   );
   const usersInSession = useMemo(
```

Comment:
isn't this the same array as the one above?

Reply:
I now re-organized the code a bit.